packages/financial-templates-lib/src/logger/Transports.ts

```diff
@@ -47,14 +50,24 @@ export function createTransports(transportsConfig: TransportsConfig = {}): Trans
     transports.push(createConsoleTransport());
   }
 
-  // If there is "test" in the environment then skip the slack and pagerduty.
+  // If there is "test" in the environment then skip the slack, pagerduty and discord.
   if (argv._.indexOf("test") == -1) {
     // If there is a slack web hook, add to the transports array to enable slack messages.
     const slackConfig: SlackConfig = transportsConfig.slackConfig ?? JSON.parse(process.env.SLACK_CONFIG || "null");
     if (slackConfig) {
       transports.push(createSlackTransport(slackConfig));
     }
 
+    // If there is a discord config, create a new transport.
+    if (transportsConfig.discordConfig || process.env.DISCORD_CONFIG) {
+      transports.push(
+        new DiscordTransport(
+          { level: "info" },
+          transportsConfig.discordConfig ?? JSON.parse(process.env.DISCORD_CONFIG || "null")
```

Comment:
Won't null cause this constructor to fail? It looks like that case isn't handled, and even if there were default args, wouldn't they only work in the `undefined` case, not `null`?

Reply:
it would not get to this point of it's undefined. this code block is only entered `if (transportsConfig.discordConfig || process.env.DISCORD_CONFIG) {`.

Also note that `JSON.parse("null")` returns undefined which would be fine with the constructor.

Reply:
Haha, of course JSON.parse("null") returns undefined.

So two points here:
1. If we never get here with both of these being undefined/null, then why do we need the || "null"? Shouldn't that be _impossible_ to hit?
2. I don't think undefined works in the constructor. I think values on `opts` are used, but undefined.[anything] will throw.

Reply:
I agree with both of your points. the net effect is we should remove the `|| "null"`. this was a carry over from the other transports. I'll remove it from all of them to be consistant.